beeline-core/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <groupId>io.honeycomb.beeline</groupId>
         <artifactId>beeline-parent</artifactId>
-        <version>1.1.0</version>
+        <version>1.1.1</version>
```

Comment:
Keep this 1.1.0 in this PR so the build works.

Reply:
As discussed in slack, I have not reverted this change.